### PR TITLE
Adding role specific details to the document educationSubmission:submit/unsubmit

### PR DESCRIPTION
**Pull request by sgoarla**
A teacher can submit or unsubmit on behalf of a student. This documentation changes is to add the expected submission state when the logged in account is as teacher or student.
The story for this change is: [removing link as this is a public repo]

Here is the current version of the docs for graph beta: 
educationSubmission: submit - https://docs.microsoft.com/en-us/graph/api/educationsubmission-submit?view=graph-rest-beta&tabs=http
educationSubmission: unsubmit - https://docs.microsoft.com/en-us/graph/api/educationsubmission-unsubmit?view=graph-rest-beta&tabs=http

As discussed, added a tabular section for roles for better readability - please review.

### CODE REVIEW

**Review by cristobal-buenrostro — APPROVED** [review]

**Comment by sgoarla** [comment]
@JarbasHorst / @Lauragra  - can one of you look at this changeset too.. Thanks

File: api-reference/beta/api/educationsubmission-unsubmit.md [hunk 1]
```diff
@@ -24,6 +24,16 @@ One of the following permissions is required to call this API. To learn more, in
...
+## Roles
+A teacher can also unsubmit a student's assignment. Below is the expected submission state, when unsubmit action is performed in a role as:
+
+| Role    | Submission state               |
+|:--------------------|:---------------------------------------------------------|
+| Teacher (work or school account)  |  working  (200 OK)  |
+| Teacher (personal Microsoft account)  |  Not supported.  |
+| Student (work or school account)  |  working (200 OK)  |
+| Student (personal Microsoft account)  |  Not supported.  |
+
```
Comment (isvargasmsft) [inline api-reference/beta/api/educationsubmission-unsubmit.md:36]: GENERAL: 

Would you please give us a bit more information about these roles? Are they formally defined roles or is it natural language? 
What permission should a student use to submit then? 

If we just need to clarify that teachers can submit on behalf of a student, I believe adding that to the first paragraph of the topic should be enough. We avoid adding additional H2s for consistency across the reference documentation.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @sgoarla, 

Please give us more context on how these roles work and what is the intention of the feature. 

Kind regards,

Isaac

File: api-reference/beta/api/educationsubmission-unsubmit.md (hunk 1 above)
Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationsubmission-unsubmit.md:36]: Hi @isvargasmsft 

the role is defined on how the user is added into the class, but for doc purposes we are using natural language.

Shanthan was following up this topic with Laura (about to use a new table or just simply mention in the paragraph. 
Her feedback was that using a table is easier to read.

You can find this conversation in the email subject "Graph documentation - Role details".
Adding screenshot for reference.

image

Comment (JarbasHorst) [reply api-reference/beta/api/educationsubmission-unsubmit.md:36]: @cristobal-buenrostro, please don't expose internal communication on GitHub. In this case, I recommend you to edit this comment and remove this image. Please, share this image with Isaac via Teams.

Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationsubmission-unsubmit.md:36]: done, sorry about that

**Review by isvargasmsft — APPROVED** [review]

File: api-reference/beta/api/educationsubmission-unsubmit.md [hunk 2]
```diff
@@ -13,7 +13,11 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Indicate that a student wants to work on the submission of the assignment after it was turned in. This action can only be taken by the student. This will change the status of the submission from "submitted" to "working". During the submit process, all the resources will be copied from submittedResources to the workingResources bucket. The teacher will be looking at the working resources list for grading.
+Indicate that a student wants to work on the submitted assignment after it was turned in. 
+
+This method changes the status of the submission from `submitted` to `working`. During the submit process, all the resources are copied from **submittedResources** to  **workingResources**. The teacher will be looking at the working resources list for grading.
```
Comment (Lauragra) [inline api-reference/beta/api/educationsubmission-unsubmit.md:18]: Should we delete the last sentence? It seems to contradict the same statement in the submit method topic. It seems to me that a teacher would not look at an "unsubmitted" resource for grading.

File: api-reference/v1.0/api/educationsubmission-unsubmit.md [hunk 3]
```diff
@@ -13,7 +13,9 @@ Namespace: microsoft.graph
 
 Indicate that a student wants to work on the submission of the assignment after it was turned in. 
 
-This action can only be taken by the student. This will change the status of the submission from "submitted" to "working". During the submit process, all the resources will be copied from submittedResources to the workingResources bucket. The teacher will be looking at the working resources list for grading.
+This method changes the status of the submission from `submitted` to `working`. During the submit process, all the resources are copied from **submittedResources** to  **workingResources**. The teacher will be looking at the working resources list for grading.
```
Comment (Lauragra) [inline api-reference/v1.0/api/educationsubmission-unsubmit.md:16]: Same comment here - delete the last sentence?

File: api-reference/beta/api/educationsubmission-unsubmit.md (hunk 2 above)
Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationsubmission-unsubmit.md:18]: Noup, teacher can grade the submission even after it was unsubmitted.

File: api-reference/v1.0/api/educationsubmission-unsubmit.md (hunk 3 above)
Comment (cristobal-buenrostro) [reply api-reference/v1.0/api/educationsubmission-unsubmit.md:16]: Noup, teacher can grade the submission even after it was unsubmitted.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**